test/data/tracker-lists.json

```diff
@@ -25,7 +25,7 @@
                 },
                 {
                     "rule": "a\\.yahoo\\.com($|[?/])",
-                    "surrogate": "yahoo.com/a.js",
+                    "surrogate": "a.js",
```

Comment:
ditto

Reply:
This changed ages ago, we're now pointing and testing the correct code with this change.